docs/source/guide/upgrading.rst

```diff
@@ -27,3 +27,11 @@ Notes to refer to when upgrading ``boto3`` versions.
     objects for transfers, use the newly added ``upload_fileobj``
     and ``download_fileobj`` methods that support both nonmultipart and
     multipart transfers.
+
+  * By default, all managed transfer methods are now threaded. In prior
```

Comment:
Yeah sounds good.  My comment was directed at changes for this specific PR, not the backfill part you added about always using threaded transfers.